### PR TITLE
Document process of addressing merge conflicts

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

This PR updates the Documentation style guide with a section on addressing merge conflicts. I also added ".DS_Store" to the gitignore file.

### CODE REVIEW

File: operations/research-and-development/product/technical-writing-team-handbook/documentation-style-guide.md [hunk 1]
```diff
@@ -696,3 +696,17 @@ Note that, longer term, it would be ideal to replace the current badge implement
...
+
+## Manage and resolve merge conflicts
+
+Mattermost hosts the source code and content of its technical documentation in GitHub repositories. Documentation is updated by submitting pull requests on GitHub. After a pull request is reviewed and approved, it is usually merged into the master branch without further changes. Sometimes, however, conflicts between the files being updated and the master branch must be resolved.
+
+Git flags conflicting regions of a file with these characters: `<<<<<<<`, `=======`, and `>>>>>>>`. The block of text between `<<<<<<<` and `=======` is the content that the pull request is attempting to merge. The block of text between `=======` and `>>>>>>>` is the block of text from the master branch that conflicts with the content in the pull request. Resolving a conflict may require removing content that has been updated in the pull request, or manually integrating text from the pull request. *The flag characters must also be removed.*
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/documentation-style-guide.md]:
```suggestion
GitHub flags conflicting regions of a file with these characters: `<<<<<<<`, `=======`, and `>>>>>>>`. The block of text between `<<<<<<<` and `=======` is the content that the pull request is attempting to merge. The block of text between `=======` and `>>>>>>>` is the block of text from the master branch that conflicts with the content in the pull request. Resolving a conflict may require removing content that has been updated in the pull request, or manually integrating text from the pull request. **The flag characters must also be removed.**
```

File: operations/research-and-development/product/technical-writing-team-handbook/documentation-style-guide.md [hunk 2]
```diff
@@ -696,3 +696,17 @@ Note that, longer term, it would be ideal to replace the current badge implement
...
+
+## Manage and resolve merge conflicts
+
+Mattermost hosts the source code and content of its technical documentation in GitHub repositories. Documentation is updated by submitting pull requests on GitHub. After a pull request is reviewed and approved, it is usually merged into the master branch without further changes. Sometimes, however, conflicts between the files being updated and the master branch must be resolved.
+
+GitHub flags conflicting regions of a file with these characters: `<<<<<<<`, `=======`, and `>>>>>>>`. The block of text between `<<<<<<<` and `=======` is the content that the pull request is attempting to merge. The block of text between `=======` and `>>>>>>>` is the block of text from the master branch that conflicts with the content in the pull request. Resolving a conflict may require removing content that has been updated in the pull request, or manually integrating text from the pull request. **The flag characters must also be removed.**
+
+After resolving the conflicts in a given file, it should be marked as resolved in GitHub. All conflicts must be resolved for the merge to proceed.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/documentation-style-guide.md]:
```suggestion
After resolving the conflicts in a given file, mark it as resolved in GitHub. All conflicts must be resolved for the merge to proceed.
```

**Review by cwarnermm — APPROVED** [review]
Thank you so much, @matthew-src! Great job!

**Comment by ghost** [comment]
Great, @cwarnermm, you are welcome!

**Pull request merged**